### PR TITLE
Revert mask creation for `plddt_labels`

### PR DESCRIPTION
**Pull request by amorehead**
* Reverts a recent change to `plddt_labels`, to ensure the construction of the `plddt_mask` follows the following criteria in the AF3 supplement:
```
`l` encompasses **all atoms** and the set of atoms `m ∈ R` is defined as:
• Atoms such that the distance in the ground truth between atom `l` and atom `m` is less than 15 Å if `m` is a protein
atom or less than 30 Å if `m` is a nucleic acid atom.
• Only atoms in polymer chains.
• One atom per token - Cα for standard protein residues and C1′ for standard nucleic acid residues.
```
In other words, the `l` indices in `plddt_mask`'s components need to store a `True` value along this dimension, so indicate that all atoms are considered as source atoms and only a subset of e.g., token center atoms are considered prospective destination atoms

### CODE REVIEW

File: alphafold3_pytorch/alphafold3.py [hunk 1]
```diff
@@ -6012,8 +6012,12 @@ def forward(
                 is_nucleotide = is_rna | is_dna
                 is_polymer = is_protein | is_rna | is_dna
 
-                is_any_nucleotide_pair = to_pairwise_mask(is_nucleotide)
-                is_any_polymer_pair = to_pairwise_mask(is_polymer)
+                is_any_nucleotide_pair = einx.logical_and(
```
Comment (lucidrains) [inline alphafold3_pytorch/alphafold3.py:6015]: ohh i see

could this and all the rest be rewritten as `repeat(is_type, '... j -> ... i j', i = is_type.shape[-1])` ?

Comment (amorehead) [reply alphafold3_pytorch/alphafold3.py:6015]: Maybe? It's hard for me to say without testing out a few inputs and outputs with this idea. The whole `torch.ones` thing I did was just to make absolutely sure that all atoms (represented as the rows in each of the constituent mask tensors) are considered when being paired up with e.g., a nucleotide atom or polymer atom, so as to not disqualify a "source atom" from being counted in `plddt_mask` just because it's a ligand or ion "source atom".

Comment (lucidrains) [reply alphafold3_pytorch/alphafold3.py:6015]: yup, just a quick sanity check!

```python
import einx
from einops import repeat
import torch

is_something = torch.randint(0, 2, (2, 16)).bool()

one = einx.logical_and('... i, ... j -> ... i j', torch.ones_like(is_something), is_something)
two = repeat(is_something, '... j -> ... i j', i = is_something.shape[-1])

assert torch.allclose(one, two)
```

Comment (amorehead) [reply alphafold3_pytorch/alphafold3.py:6015]: Cool! Will push in a new PR

Comment (lucidrains) [reply alphafold3_pytorch/alphafold3.py:6015]: hehe not a big deal

my bad for not understanding the intent!
